core/templates/dev/head/App.js

```diff
@@ -24,7 +24,8 @@ var oppia = angular.module(
     'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate', 'toastr',
-    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate', 'topicLandingPageModule',
+    'learnerDashboardPageModule'
```

Comment:
Move the `learnerDashboardPageModule` before `topicLandingPageModule` so that it is alphabetized.

Reply:
done